src/components/graph/network-modification-tree-model.js

```diff
@@ -97,7 +110,24 @@ export default class NetworkModificationTreeModel {
                     });
                 });
             });
-            this.treeElements = filteredTreeElements;
+
+            // fix parentNodeUuid of children
+            const willbeDeletedNode = this.treeElements.find(
```

Comment:
if it's possible to rename this cont willbeDeletedNode => nodeToDelete or same but we respect the camel case willbeDeletedNode => willBeDeletedNode